lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -57,9 +57,17 @@ sub show : Chained('load') PathPart('')
     my ($self, $c) = @_;
 
     my $cdtoc = $c->stash->{cdtoc};
+    my $medium_cdtocs = $self->_load_releases($c, $cdtoc);
+
+    for my $mc (@$medium_cdtocs) {
+        ${$mc}{perfect_match} = $c->model('Medium')->perfect_match_cdtoc(
```

Comment:
Or, maybe you'll agree with making `perfect_match` a method on `Entity::MediumCDTOC`.
 (to replace `Data::Medium::perfect_match_cdtoc`) as described in https://github.com/metabrainz/musicbrainz-server/pull/504#discussion_r111528011, in which case this could be removed.